README.rst

```diff
@@ -19,15 +19,15 @@ Using
 
 Once installed, you can run a PyPI server with::
 
-   pypi-timemachine 2014-02-03
+   pypi-timemachine 2021-02-03
 
 or if you need to specify a precise time (in UTC)::
 
-   pypi-timemachine 2014-02-03T12:33:02
+   pypi-timemachine 2021-02-03T12:33:02
 
-This will start up a Flask app, and will print out a line such as::
+This will start up a webapp running uvicorn, and will print out a line such as::
 
-   Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
+   Starting pypi-timemachine server at http://127.0.0.1:5000/
```

Comment:
Very minor, but this message doesn't indicate whether the server is actually ready to receive connections or whether it is about to start (compared to the old message).